addons/website/static/src/scss/primary_variables.scss

```diff
@@ -255,6 +255,7 @@ $o-base-website-values-palette: (
     'btn-secondary-outline': false,
     'header-template': 'logo_menu',
     'footer-template': 'products_contacts_about_us',
+    'footer-effect': null, // null / 'slideout_slide_hover' / 'slideout_shadow'
```

Comment:
I added this line, don't forget that any website variable you allow the user to customize should be described in this map